packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -487,7 +487,9 @@ export class EpochTracker implements IPersistedFileCache {
 	}
 
 	private fileEntryFromEntry(entry: IEntry): ICacheEntry {
```

Comment:
it seems like this could be pulled out into a free function, and then you can easily unit test it